libtransmission/jsonsl.c

```diff
@@ -792,7 +792,7 @@ populate_component(char *in,
     }
 
     /* Replace the next / with a NULL */
-    *next = strstr(in, "/");
+    *next = strchr(in, '/');
```

Comment:
You're right, but unless there's a serious bug in jsonsl.c I'd rather leave it as-is for consistency with upstream. You can PR this at https://github.com/mnunberg/jsonsl, though that repo seems to not be actively maintained.

TBH the real solution would be to replace jsonsl.c with nlohmann or rapidjson or simdjson, but doing that during the 4.0.0 beta cycle is probably not a great idea: :stuck_out_tongue:

Reply:
@ckerr Okay now I'll open them PR

Reply:
> Did you use a tool to find these changes, or was it through manual code inspection?

Obviously tool and thorough signature checks. Coverity Scan helped creation this PR.